src/main/java/com/github/dockerjava/api/model/Statistics.java

```diff
@@ -19,6 +21,17 @@
     @JsonProperty("read")
     private String read;
 
+    /**
+     * @since Docker Remote API 1.21
+     */
+    @CheckForNull
+    @JsonProperty("networks")
+    private Map<String, Object> networksStats;
```

Comment:
You can rework it as part of PR imho, 3.0 is not released and allows doing API breakage.